bot/pagination.py

```diff
@@ -99,7 +99,7 @@ async def paginate(
         timeout: int = 300,
         footer_text: str = None,
         url: str = None,
-        exception_on_empty_embed: bool = False,
+        exception_on_empty_embed: bool = False
```

Comment:
Don't do this. Trailing commas are good design, because if we were to add another kwarg we only have to change a single line instead of two, which ensures correct git blame is maintained.